packages/expo-updates/android/src/main/java/expo/modules/updates/UpdatesModule.java

```diff
@@ -217,6 +217,7 @@ public void onSuccess(@Nullable UpdateEntity update) {
                 if (update == null) {
                   updateInfo.putBoolean("isNew", false);
                 } else {
+                  updatesService.resetSelectionPolicy();
```

Comment:
Do we want to keep the selection policy when there is an error? What behavior lines up best with the intended use of `setSelectionPolicy`?

Reply:
yeah -- the intent here is to keep the existing selection policy (which may always choose to load a single predefined update) unless the developer or user has specifically downloaded a new update programatically using `fetchUpdateAsync`; in this case they would expect to see this new update upon reloading. But in any other case, including an error, keeping the existing policy is intended.